app/Models/Bible/BibleBook.php

```diff
@@ -22,14 +24,14 @@ class BibleBook extends Model
     protected $connection = 'dbp';
     protected $table = 'bible_books';
     public $incrementing = false;
-    public $fillable = ['abbr','book_id', 'name', 'name_short', 'chapters'];
+    public $fillable = ['abbr','book_id', 'name', 'name_short', 'chapters', 'bible_seq'];
     public $hidden = ['created_at','updated_at','bible_id'];
 
 
     /**
      *
      * @OA\Property(ref="#/components/schemas/Bible/properties/id")
-     * @method static BibleFileset whereBibleId($value)
```

Comment:
are you just fixing incorrect comments here?

Reply:
@bradflood Yes, I am. It is a fix about comments.